tests/test_cli.py

```diff
@@ -349,10 +349,8 @@ def test_readme_command(self):
             self._main('buildreadme')
             # README and spec are opened
             self.assertEqual(mock_file.call_args_list[0][0],
-                             ('README.md',))
```

Comment:
This file would only open initially if it already exists in the path. This test has no `README.md` file initially created, therefore the only two files opened in order are `spec.json` and `README.md` with the write flag

Reply:
Never mind, just happening on my machine